src/components/about/About.test.tsx

```diff
@@ -0,0 +1,92 @@
+import * as React from "react";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import About from "./About";
+import { Router } from "react-router-dom";
+import { createMemoryHistory } from "history";
+
+test("should render component", () => {
+  const history = createMemoryHistory({
+    initialEntries: ["/about"],
+  });
+  render(
+    <Router history={history}>
+      <About />
+    </Router>
+  );
```

Comment:
Minor thing, but I found the `Router` and `history` weren't actually required unless you were clicking the links so you might be able to simplify this test (and possibly others) a bit if you wanted to with:

```suggestion
  render(<About />);
```

Reply:
I doubt as I tried removing router, but the component itself uses `withRouter` HOC which will force me to wrap my component in jest otherwise my test will fail,

`Invariant failed: You should not use <withRouter(About) /> outside a <Router>`
This is the error that I'm getting if I don't wrap it with router. This might not be required once we start the refactoring phase but I'm not 100% sure right now